solution/requirements.txt

```diff
@@ -0,0 +1,6 @@
+langchain
```

Comment:
Why do we have this twice?

Reply:
a remnant from development, going to remove this file.